datetime/src/test/scala/com/fortysevendeg/scalacheck/datetime/jdk8/GenJdk8Properties.scala

```diff
@@ -43,7 +40,21 @@ object GenJdk8Properties extends Properties("Java 8 Generators") {
 
   property("arbitrary generation creates valid times (with no granularity)") = {
     import ArbitraryJdk8._
-    forAll { dt: ZonedDateTime =>
+    forAll { _: ZonedDateTime =>
+      passed
+    }
+  }
+
+  property("arbitrary generation creates valid times (with no granularity)") = {
+    import ArbitraryJdk8._
+    forAll { _: LocalDateTime =>
+      passed
+    }
+  }
+
+  property("arbitrary generation creates valid times (with no granularity)") = {
+    import ArbitraryJdk8._
+    forAll { _: Instant =>
```

Comment:
it's missing a spec for `LocalDate`, no?

Reply:
Good catch, fixed in 72cd231